intro-HPC/ch_mympirun.tex

```diff
@@ -0,0 +1,72 @@
+\chapter{Mympirun}
+\label{ch:mympirun}
+
+\lstinline|mympirun| is a tool to make it easier for users of HPC clusters to run
+MPI programs with good performance. We strongly recommend to use \lstinline|mympirun|
+instead of \lstinline|impirun|.
+
+In this chapter, we give a high-level overview. For a more detailed description of all
+options, see the
+\href{https://github.com/hpcugent/vsc-mympirun/blob/master/README.md}{vsc-mympirun README}.
+
+\section{Basic usage}
+\label{sec:myrun-basic-usage}
+
+The most basic form of using \lstinline|mympirun| is \lstinline|mympirun [mympirun options] your_program [your_program options]|.
+
+For example, to run a program named \lstinline|hello_mpi| and give it a single argument (\lstinline|5|),
+we can run it with \lstinline|mympirun ./hello_mpi 5|.
```

Comment:
Don't use `./` here, it'll confuse people. Usually people run provided software after loading a module, which doesn't require `./`.

`mympirun example 5`